src/extension.ts

```diff
@@ -132,18 +132,19 @@ export async function activate(context: ExtensionContext): Promise<void> {
       try {
         const javaHome = await getJavaHome(getJavaHomeFromConfig());
         await fetchAndLaunchMetals(context, javaHome, serverVersion);
-        await showReleaseNotes(
-          "onExtensionStart",
-          context,
-          serverVersion,
-          outputChannel
-        );
       } catch (err) {
         outputChannel.appendLine(`${err}`);
         showMissingJavaMessage();
       }
     }
   );
+
+  await showReleaseNotes(
+    "onExtensionStart",
+    context,
+    serverVersion,
+    outputChannel
+  );
```

Comment:
Ok, moving this out of `window.withProgress` should solve "Starting Metals server..." forever issue, but it's unrelated to the timeout thing.